.nvmrc

```diff
@@ -0,0 +1 @@
+18
```

Comment:
added .nvmrc because it's been bugging me that I can't automatically switch to the version of node that's required to build thisi project